bashflux/bashflux.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"log"
+
+	"github.com/mainflux/mainflux/bashflux/cmd"
+	"github.com/spf13/cobra"
+)
+
+type Config struct {
```

Comment:
There is no need for exporting this. Please remove it, and use as an anonymous structure inside `main()`.